test/e2e/query_test.go

```diff
@@ -552,3 +612,55 @@ func rangeQuery(t *testing.T, ctx context.Context, addr string, q string, start,
 		return errors.Errorf("unexpected results size %d", len(res))
 	}))
 }
+
+func TestQueryMultiTenancy(t *testing.T) {
+	t.Parallel()
+
+	s, err := e2e.NewScenario("e2e_test_query_mul")
+	testutil.Ok(t, err)
+	t.Cleanup(e2ethanos.CleanScenario(t, s))
+
+	receiver1, err := e2ethanos.NewReceiver(s.SharedDir(), s.NetworkName(), "1", 1)
```

Comment:
We need to add in this also for checking if creation of receiver was successful.
```suggestion
	receiver1, err := e2ethanos.NewReceiver(s.SharedDir(), s.NetworkName(), "1", 1)
	testutil.Ok(t, err)
```